larq_compute_engine/mlir/python/end2end_test.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import pytest
+import larq as lq
+import subprocess
+import tensorflow as tf
+import numpy as np
+from tensorflow.python.eager import context
+
+from larq_compute_engine.mlir.python.converter import convert_keras_model
+
+
+@pytest.fixture
+def eager_mode():
+    """pytest fixture for running test in eager mode"""
+    with context.eager_mode():
+        yield
+
+
+def testnet():
```

Comment:
If you make this a fixture it will be passed into the test directly:
```suggestion
@pytest.fixture
def model():
```